src/components/footer/data.ts

```diff
@@ -41,35 +41,35 @@ const resources = {
   ],
 };
 
-const top_adopters = {
-  title: "Top Adopters",
-  links: [
-    {
-      name: "Okteto",
-      url:
-        "https://github.com/litmuschaos/litmus/blob/master/adopters/okteto.md",
-    },
-    {
-      name: "Intuit",
-      url: "https://github.com/litmuschaos/litmus/blob/master/ADOPTERS.md",
-    },
-    {
-      name: "Wipro",
-      url:
-        "https://github.com/litmuschaos/litmus/blob/master/adopters/AppAnywhere.md",
-    },
-    {
-      name: "OpenEBS",
-      url:
-        "https://github.com/litmuschaos/litmus/blob/master/adopters/openebs.md",
-    },
-    {
-      name: "Zebrium",
-      url:
-        "https://github.com/litmuschaos/litmus/blob/master/adopters/zebrium.md",
-    },
-  ],
-};
+// const top_adopters = {
+//   title: "Top Adopters",
+//   links: [
```

Comment:
are these not needed anymore?

Reply:
No, these commented lines will be removed. The design requirements have been updated.